tests/examples/wibox/awidget/prompt/hooks.lua

```diff
@@ -1,7 +1,8 @@
 local parent    = ... --DOC_NO_USAGE --DOC_HIDE
 local wibox     = require( "wibox"     ) --DOC_HIDE
 local awful     = { prompt = require("awful.prompt"),--DOC_HIDE
-                    util   = require("awful.util")}--DOC_HIDE
+                    util   = require("awful.util"),
```

Comment:
Not really, it's still unrelated to the commit. Spawn is never *used* in the test. While the example code has it, it's never called. Also, it should not be called in documentation test. We currently have no way to shim the commands.